diff --git a/node_modules/hexo-theme-icarus/layout/widget/qq-group.jsx b/node_modules/hexo-theme-icarus/layout/widget/qq-group.jsx
new file mode 100644
index 0000000..455a8b6
--- /dev/null
+++ b/node_modules/hexo-theme-icarus/layout/widget/qq-group.jsx
@@ -0,0 +1,32 @@
+const { Component } = require('inferno');
+const { cacheComponent } = require('hexo-component-inferno/lib/util/cache');
+
+class QqGroup extends Component {
+    render() {
+        const { imgUrl, linkUrl, title } = this.props;
+        return <div class="card widget" data-type="qq-group">
+            <div class="card-content">
+                <div class="menu-label">
+                    {title}
+                </div>
+                <div style="text-align: center">
+                    <a target="_blank" rel="noopener" href={linkUrl}>
+                        <img src={imgUrl} alt={title} title={title} style="max-width: 100%" />
+                    </a>
+                </div>
+            </div>
+        </div>;
+    }
+}
+
+QqGroup.Cacheable = cacheComponent(QqGroup, 'widget.qq-group', props => {
+    const { widget } = props;
+    const { url_for } = props.helper;
+    return {
+        imgUrl: url_for(widget.img_url),
+        linkUrl: widget.link_url,
+        title: widget.title || 'PowerShell 技术 QQ 群'
+    };
+});
+
+module.exports = QqGroup;
